src/handlers/notification-banner-handler.ts

```diff
@@ -0,0 +1,29 @@
+import { RequestHandler } from "express";
+import { getParameter } from "../services/parameterStoreService";
+
+const notificationBannerHandler: RequestHandler = async (req, res, next) => {
+  try {
+    const data = await getParameter("notification-banner");
+    res.locals.displayBanner = false;
+    if (!data) {
+      return next();
+    }
+
+    const currentTime = Date.now();
+    if (
+      req.path === data.pageId &&
+      currentTime >= data.startTime &&
+      currentTime <= data.endTime
+    ) {
+      res.locals.displayBanner = true;
+      res.locals.bannerType = data.bannerType;
```

Comment:
What is `bannerType`?

Reply:
Its parameter in banner component, if we want to display green success banner we can pass it from here.